### PR TITLE
PRO-961 WalletConnect Session request UI

### PR DESCRIPTION
**Pull request by hemadri-tank-bacancy**
* Points
   1. Made WalletConnect Session request UI
   2. Implemented different chainId approve in wallet connect.

https://user-images.githubusercontent.com/78720269/213160464-9a0cb3c8-4710-40d5-9a94-fd5566e52407.mp4

### CODE REVIEW

File: src/locales/en/common.json [hunk 1]
```diff
@@ -1119,6 +1120,12 @@
     "error": {
       "smartWalletNeedToBeActivated": "Smart Wallet needs to be activated to proceed.",
       "unsupportedRequestCallRequestType": "Unsupported Call Request type."
+    },
+    "paragraph":{
+      "undeploy_warning_1": "You are connecting to a wallet that&apos;s undeployed, you will not be able to execute any transaction, go to ",
```
Comment (IAmKio) [inline src/locales/en/common.json:1125]: The aopostrophe html entity code is literally printed in the UI (check the video)

File: src/screens/WalletConnect/Requests/WalletConnectSwitchNetwork.tsx [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+      </View>
+
+      <ListModal />
+    </>
+  );
+};
+
+export default WalletConnectSwitchNetwork;
+
+const useChains = (): any[] => {
```
Comment (IAmKio) [inline src/screens/WalletConnect/Requests/WalletConnectSwitchNetwork.tsx:209]: Could we please move this function to the top of the render method for consistency

**Review by IAmKio — CHANGES_REQUESTED** [review]
A few small things but other than that - great work

File: src/locales/en/common.json (hunk 1 above)
Comment (hemadri-tank-bacancy) [reply src/locales/en/common.json]: Fixed.

File: src/screens/WalletConnect/Requests/WalletConnectSwitchNetwork.tsx (hunk 2 above)
Comment (hemadri-tank-bacancy) [reply src/screens/WalletConnect/Requests/WalletConnectSwitchNetwork.tsx]: Updated.

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**